packages/http-client/src/remote-admin-api.ts

```diff
@@ -43,16 +49,20 @@ export class RemoteAdminApi implements AdminApi {
 
   async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
     const code = await this.generateCode()
+    const did = this._getDidFn()
+    if (!did) throw new MissingDIDError()
```

Comment:
would it be easier to put this check into `buildJWS` so the check can just live in one place?

Reply:
yeah i thought about that, originally i only had it here and not in the other two calls - yeah lemme move it

Reply:
ok done